packages/itwinui-react/src/core/Header/HeaderLogo.test.tsx

```diff
@@ -38,34 +38,30 @@ it('renders with onClick correctly', () => {
     </HeaderLogo>,
   );
 
-  const root = container.querySelector('.iui-header-brand') as HTMLDivElement;
-  expect(root).toBeTruthy();
-  expect(root.getAttribute('role')).toBe('button');
+  const root = container.querySelector('button') as HTMLButtonElement;
+  expect(root).toHaveClass('iui-header-brand');
   root.click();
 
   expect(onClickMock).toHaveBeenCalled();
 });
 
-it('handles keypress with onClick correctly', () => {
+it('renders with as prop correctly', () => {
   const onClickMock = jest.fn();
   const { container } = render(
-    <HeaderLogo logo={<SvgPlaceholder />} onClick={onClickMock}>
-      Application
+    <HeaderLogo
+      as='a'
```

Comment:
In this case clicking on item opens a link or does onClick action? Or both?

Reply:
both. `onClick` is in addition to the default action